include/kernel.h

```diff
@@ -4224,6 +4224,7 @@ struct k_mem_slab {
 	char *free_list;
 	uint32_t num_used;
 
+	bool is_out_of_mem;
```

Comment:
It this new field actually needed? Couldn't `free_list == NULL` be checked instead?

Reply:
Changed.